apps/web-giddh/src/app/theme/ngx-date-range-picker/ngx-daterangepicker.component.scss

```diff
@@ -1074,15 +1163,493 @@ td span {
+        flex: 1 1 auto;
+        // overflow: auto;
+        .calender-body {
+            display: flex;
+            flex-direction: column;
+            .upper-part{
+                border-bottom: 1px solid $white-smoke;
+                position: sticky;
+                top: 0;
+                z-index: 1;
+                background-color:$white;
+                .sticky-header{
+                    width:calc(100% - 3px) !important;
+                    th.days-td {
+                        position: sticky;
+                        top: 100px;
+                        z-index: 99;
+                        
+                    }
+                }
+                .calendar-table{
+                    padding-top: 10px;
+                }
+            }
+            .select-date-range {
+                display: flex;
+                justify-content: space-between;
+                padding: 10px 10px 0;
+                input {
+                    width: 100px;
+                    height: auto;
+                    font-size: 12px;
+                    padding: 6px 0;
+                    border: none;
+                    border-bottom: 1px solid $gainsboro;
+                    &:focus {
+                        outline: none;
+                        box-shadow: none;
+                    }
+                }
+
+                span {
+                    display: flex;
+                    align-items: flex-start;
+                }
+                .activeSelectedDate {
+                    border-bottom-color: $primary !important;
+                    box-shadow: 0 1px 0 0 $primary !important;
+                    &.invalid-date-input{
+                        border-bottom-color:$red !important;
+                        box-shadow: 0 1px 0 0 $red !important;
+                        
+                    }
+
+                }
+                .invalid-date-input{
+                    border-bottom-color:$red !important;
+                    box-shadow: 0 1px 0 0 $red !important;
+                    
+                }
 
-.md-drppicker .calendar th, .md-drppicker .calendar td {
-    height: 30px;
-    border: 1px dashed transparent;
+            }
+        }
+    }
+}
+
+.month-field {
+    display: flex;
+    justify-content: space-between;
+    padding:16px 10px 2px;
+         .month-dropdown{
+        display: flex;
+        flex-direction: row;
+        width: 180px;
+        justify-content: flex-start;
+        align-items: center;
+        font-size: 13px;
+        .prev.available{
+            display: inline-flex;
+            transform: rotate(270deg);
+            margin-left: 6px;
+        }
+    }
+    .month-name {
+        font-weight: 700;
+        cursor: pointer;
+        font-style: initial;
+        display: flex;
+    }
+}
+.prev-next-btn {
+    width: 100px;
+    display: flex;
+    justify-content: space-around;
+}
+.days-upto-select {
+    border-bottom: 1px solid gainsboro;
```

Comment:
Incorrect use of SASS variables (missing $)

Reply:
updated